beta-architecture/beta-architecture.md

```diff
@@ -0,0 +1,32 @@
+# Beta Architecture
+
+## Initial Design
+
+The initial architectural design for beta is shown below. 
+
+This is not the final architecture, and it will change during the course of the beta phase of the TDR project.
+
+![](./diagrams/tdr-beta-high-level-architecture.svg)
+
+#### AWS Accounts
+
+AWS accounts will be set up for each environment that will be used.
+
+#### Environments
+
+Multiple environments will be used:
+* Continuous Integration (CI)
+* Test
+* Production
+
+#### TDR
+
+The application stack will run within the defined environments. 
+
+### Assumptions
+
+[TODO]
+
+### Open Questions
+
+[TODO]
```

Comment:
There's a big open question about what auditing is needed, since it might add more components to the architecture.

There are open questions around how we use the SNS topics (and whether we use them at all).

Reply:
Added these open questions